pyproject.toml

```diff
@@ -258,6 +257,8 @@ min-similarity-lines = 12
 # Add the specified OPTS to the set of command line arguments as if they had been
 # specified by the user.
 addopts = "--typeguard-packages=paperqa"
+# set how loops are scoped to avoid https://github.com/BerriAI/litellm/issues/5854
+asyncio_default_fixture_loop_scope = "session"
```

Comment:
Confirmed this worked in https://github.com/Future-House/paper-qa/pull/674

Nice work here @whitead, we are unblocked again